lib/galaxy/managers/jobs.py

```diff
@@ -662,3 +662,24 @@ def inputs_recursive(input_params, param_values, depth=1, upgrade_messages=None)
 
     parameters = inputs_recursive(tool.inputs, params_objects, depth=1, upgrade_messages=upgrade_messages)
     return {"parameters": parameters, "has_parameter_errors": has_parameter_errors}
+
+
+def summarize_job_information(trans, job):
+    """Produce a dict-ified version of job information
```

Comment:
Can you use the existing endpoint in api/jobs/job_id and do a separate call to decode ids for admins?

Reply:
@mvdbeek 
So, I tried and I am missing `copied_from_job_id` and `toold_id`

Reply:
I think it's fine to add them to the full_details view

Reply:
@mvdbeek 
thanks a lot for helping me offline